### DISCUSSION TITLE
Security vulnerability in d3-interpolate

### DISCUSSION THREAD

**Discussion by azhirnova** [recharts/recharts#2905] [General]
Hi there, there is a [security vulnerability](https://security.snyk.io/vuln/SNYK-JS-D3COLOR-1076592) in the d3-color package (dependency of d3-interpolate) versions before 3.1.0. I was wondering if you were planning to bump d3-interpolate to the latest version any time soon?

**Comment by rodoabad** [comment]
Bumping this one since it's being reported now as a high vulnerability.

**Reply by ckifer** [reply]
Please see discussion in #3012

**Reply by rodoabad** [reply]
Thanks, @ckifer !

**Reply by rodoabad** [reply]
I'm seeing this in the alpha release as well - https://github.com/recharts/recharts/blob/2.3.0-alpha.x/package.json#L89 where we're using the new version.

**Reply by ckifer** [reply]
@rodoabad yeah I'm trying to do an alpha release to test that this doesn't break things, however I think `alpha.0` left the vulnerability fix out.

I do not have publish permissions so this has been tricky. I will try to get `alpha.1` published tonight that will have the fix. Its reliant on a response and action from the owner though

**Comment by ckifer** [comment]
https://www.npmjs.com/package/recharts/v/2.3.0-alpha.1 

please help me test this! Thanks!

**Reply by rodoabad** [reply]
Thanks for the update @ckifer, we'll try and test this tomorrow for ya!

**Reply by rodoabad** [reply]
@ckifer tested it in our applications and it works for us.

**Reply by rodoabad** [reply]
Automated tests pass, manual tests passed, we're confident in this version as the stable version.

**Reply by ckifer** [reply]
Awesome, much appreciated!

Going to wait a few more days or so to let others test and then release 2.3.
